<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveVariableEditor</class>
 <widget class="QDialog" name="SaveVariableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable Editor</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Name</string>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>T&amp;ype</string>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbType">
     <item>
      <property name="text">
       <string>Bool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Number</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblValue">
     <property name="text">
      <string>&amp;Value</string>
     </property>
     <property name="buddy">
      <cstring>cbValue</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leValue"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbValue">
       <item>
        <property name="text">
         <string>False</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>True</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbValue">
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveVariableEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SaveVariableEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbType</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SaveVariableEditor</receiver>
   <slot>updateType(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateType(QString)</slot>
 </slots>
</ui>
